jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFDependencyProcessor.java

```diff
@@ -156,7 +156,7 @@ private void addJSFInjection(String jsfVersion, ModuleSpecification moduleSpecif
             } else {
                 JSFLogger.ROOT_LOGGER.loadingJsf2x();
                 jsfInjectionDependency.addImportFilter(PathFilters.getMetaInfFilter(), true);
-                // Exclude JSF 1.2 faces-config.xml to make extra sure it won't interfere with JSF 2.0 deployments
+                // Exclude Faces 1.2 faces-config.xml to make extra sure it won't interfere with JSF 2.0 deployments
```

Comment:
I think the original `JSF 1.2` comment was the best we can do.  @fjuma do you agree?

Reply:
Yes.